api/soap/mantisconnect.wsdl

```diff
@@ -163,6 +163,7 @@
    <xsd:element name="monitors" type="tns:AccountDataArray" minOccurs="0"/>
    <xsd:element name="sticky" type="xsd:boolean" minOccurs="0"/>
    <xsd:element name="tags" type="tns:ObjectRefArray" minOccurs="0"/>
+   <xsd:element name="token" type="xsd:string" minOccurs="0"/>
```

Comment:
The URL is either detected from the URL used to access the API or is fixed as part of the configuration, so I'm not sure this is a reason to just return the token.  Returning the full URL allows the URL to be opaque to the client and enable changing how such URL works and what script it points to without changing the clients.

We can also consider whether this should be a separate API to get the share URL instead of this being part of the normal issue get method.  That also enables us to separate the code into a plugin if we decide to do so.